modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlAsynchronousApiTest.java

```diff
@@ -597,6 +597,19 @@ public void errors() {
             await(ars.closeAsync());
             assertThrowsWithCause(() -> await(ars.fetchNextPage()), CursorClosedException.class);
         }
+
+        // DDL is non-transactional.
```

Comment:
I would like an additional test when we do a few operations in the transaction and just after it calls DDL in the same TX. Check that transaction is not rolled back or committed